spec/jobs/get_usps_proofing_results_job_spec.rb

```diff
@@ -744,7 +718,6 @@
             expect(job_analytics).to have_logged_event(
               'GetUspsProofingResultsJob: Unexpected response received',
               hash_including(
-                minutes_since_established: range_approximating(3.days.in_minutes, vary_right: 5),
                 reason: 'Invalid enrollment code',
```

Comment:
```suggestion
                reason: 'Invalid enrollment code',
                response_message: /Enrollment code [0-9]{16} does not exist/,
```
could also test for response message here